### PR TITLE
Fair card updates

### PR DESCRIPTION
**Pull request by ashfurrow**
This change brings in the hero image as the first image in the card:

Screen Shot 2020-03-05 at 15 10 49

Also includes a changelog entry for #3041.

### CODE REVIEW

File: src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx [hunk 1]
```diff
@@ -12,14 +13,15 @@ const artworkNode = {
     image: { url: "https://example.com/image.jpg" },
   },
 }
-const fairsModule = {
+const fairsModule: Omit<FairsRail_fairs_module, " $refType"> = {
```
Comment (ashfurrow) [inline src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx]: `Omit` is a really fun TypeScript generic. It lets us use the `FairsRail_fairs_module` type generated by Relay, but omit the `" $refType"` field.

File: src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx [hunk 2]
```diff
@@ -57,8 +60,8 @@ it("looks correct when rendered", () => {
 it("looks correct when rendered with fairs missing artworks", () => {
   const fairsCopy = cloneDeep(fairsModule)
   fairsCopy.results.forEach(result => {
-    result.followedArtistArtworks.edges = []
-    result.otherArtworks.edges = []
+    ;(result.followedArtistArtworks as any).edges = []
+    ;(result.otherArtworks as any).edges = []
```
Comment (ashfurrow) [inline src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx]: Unfortunately, the `FairsRail_fairs_module` type specifies `edges` is `readonly`, so I cast `as any` here to opt out of type checking for this line.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/eigen/blob/fair-card-updates/src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx">FairsRail-tests.tsx</a>


<li>
<a href='https://github.com/artsy/eigen/blob/fair-card-updates/src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx#L57'>looks correct when rendered</a>
<br/>
Snapshot name: `looks correct when rendered 1`- Snapshot+ Received@@ -123,11 +123,11 @@ }, }, ], }, "profile": Object {- "url": "https://neopets.com",+ "slug": "https://neopets.com", }, "slug": "the-fair", }, Object { "exhibitionPeriod": "Monday–Friday",

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(received).toMatchSnapshot()

Snapshot name: `looks correct when rendered 1`

- Snapshot
+ Received

@@ -123,11 +123,11 @@
                  },
                },
              ],
            },
            "profile": Object {
-             "url": "https://neopets.com",
+             "slug": "https://neopets.com",
            },
            "slug": "the-fair",
          },
          Object {
            "exhibitionPeriod": "Monday–Friday",
    at Object.toMatchSnapshot (/Users/travis/build/artsy/eigen/src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx:57:16)
    at Object.asyncJestTest (/Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at /Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/queueRunner.js:43:12
    at new Promise (<anonymous>)
    at mapper (/Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at /Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/queueRunner.js:73:41
</code></pre>
</details>
</li>
<br/>

</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx (hunk 2 above)
Comment (ds300) [reply src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx]: another option here is to add a line above with `// @ts-ignore`

Comment (ashfurrow) [reply src/lib/Scenes/Home/Components/ForYou/Components/__tests__/FairsRail-tests.tsx]: Oh I like that much more, thanks 👍

**Pull request merged**